src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -13,12 +16,15 @@ public class ListCommand extends ViewCommand {
     public static final String COMMAND_WORD = "view";
 
     public static final String MESSAGE_SUCCESS = "Viewing all students";
+    private static final Logger logger = Logger.getLogger(ListCommand.class.getName());
```

Comment:
Same thing here

Reply:
Thanks for the catch! I fixed them to be the same other other classes, will merge them in.